src/Request/StreamExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Botwin.Request
+{
+    using System.IO;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public static class StreamExtensions
+    {
+        public static string AsString(this Stream stream)
+        {
+            var reader = new StreamReader(stream);
+
+            var readStream = reader.ReadToEnd();
+
+            stream.Position = 0;
```

Comment:
ASP.Net Core body can be a couple of types, therefore it's best to do `if {stream.CanSeek}{stream.Position=0}